src/main/kotlin/ru/meanmail/quickfix/InstallPackageQuickFix.kt

```diff
@@ -21,14 +22,11 @@ class InstallPackageQuickFix(element: NameReq,
                         endElement: PsiElement) {
         val element = (startElement as? NameReq) ?: return
         val packageName = element.name.text ?: return
-        val versionOne = element.versionspec?.versionMany?.versionOneList?.get(0) ?: return
-        val versionCmp = versionOne.versionCmp.text ?: "=="
 
-        installPackage(project, packageName, version, versionCmp) {
+        installPackage(project, packageName, requirement) {
             val application = ApplicationManager.getApplication()
             application.invokeLater {
                 application.runWriteAction {
-                    versionOne.setVersion(version)
```

Comment:
This is a very nice feature :cry:. It should be fixed and left

Reply:
What does this do exactly?

Reply:
After updating the package, also in the requirements.txt file fixes the required version of the package

Reply:
I fixed it. The version is updated in requirements.txt when upgrading the version (quick fix on "outdated" message). I don't update it when installing in from zero because then conditions like `==1.*` or `>=3.0,<4.0` would be overridden.